### PR TITLE
FailoverClustering Bulk Fixup (2/4) - Line formatting updates

### PR DESCRIPTION
**Pull request by robinharwood**
Bulk reflowed text and header formatting update before applying further quality updates.

This will be followed by separate PRs for 
	1. Text formatting updates
		a. Bulk formatting checklist update before applying further quality updates:
			- cmdlet name - Change bolds to code spans
			- cmdlet/vars - Change bar ecmdlet/var to code spans
			- parameters - change italic formatting to bold
			- parameters - change code span formatting to bold
			- change italics to use underscores
			- change bold-underscores to asterisk
			- change bold keystrokes to kbd
			- Use contractions
			- Fix indented code blocks
			- Find bad syntax reflows
	2. Update example formatting
		a. Bulk updating example formatting before content updates

### CODE REVIEW

**Comment by robinharwood** [comment]
Hi @michaeltlombardi / @sdwheeler,

I believe this PR is release ready if you would be kind enough to review and if appropriate sign-off/merge 😊

The main update was to reflow and fix any line return issues.

Please let me know if you have any questions or if I need to apply any other updates.

Thanks Robin

File: docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md [hunk 1]
```diff
@@ -145,19 +158,23 @@ Accept wildcard characters: False
...
 
 ### -MigrationType
-Specifies the type of migration to perform for the virtual machine.
-The options are as follows: 
 
- -- Live: Transparently migrates the virtual machine without a dropped network connection or perceived downtime. 
+Specifies the type of migration to perform for the virtual machine. The options are as follows:
+
+ -- Live: Transparently migrates the virtual machine without a dropped network connection or
+ perceived downtime.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md:165]: This suggestion (and the following) turn the non-standard syntax into a list - I would've done them in a single suggestion but the unedited lines in between made GitHub unhappy.

```suggestion
- `Live`: Transparently migrates the virtual machine without a dropped network connection or
  perceived downtime.
```

File: docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md [hunk 2]
```diff
@@ -145,19 +158,23 @@ Accept wildcard characters: False
...
 
- -- Quick: Rapidly migrates a running virtual machine with minimal downtime. 
+ -- Quick: Rapidly migrates a running virtual machine with minimal downtime.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md:167]:
```suggestion
- `Quick`: Rapidly migrates a running virtual machine with minimal downtime.
```

File: docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md [hunk 3]
```diff
@@ -145,19 +158,23 @@ Accept wildcard characters: False
...
 
- -- Shutdown: Performs an orderly shutdown of the operating system (waiting for all processes to close) on the virtual machine, and then migrates the virtual machine. 
+ -- Shutdown: Performs an orderly shutdown of the operating system (waiting for all processes to
+ close) on the virtual machine, and then migrates the virtual machine.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md:170]:
```suggestion
- `Shutdown`: Performs an orderly shutdown of the operating system (waiting for all processes to
  close) on the virtual machine, and then migrates the virtual machine.
```

File: docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md [hunk 4]
```diff
@@ -145,19 +158,23 @@ Accept wildcard characters: False
...
 
- -- ShutdownForce: Shuts down the operating system on the virtual machine without waiting for slower processes to finish, and then migrates the virtual machine. 
+ -- ShutdownForce: Shuts down the operating system on the virtual machine without waiting for slower
+ processes to finish, and then migrates the virtual machine.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md:173]:
```suggestion
- `ShutdownForce`: Shuts down the operating system on the virtual machine without waiting for slower
  processes to finish, and then migrates the virtual machine.
```

File: docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md [hunk 5]
```diff
@@ -145,19 +158,23 @@ Accept wildcard characters: False
...
 
- -- TurnOff: Turns off the virtual machine without shutting down the operating system first, then migrates the virtual machine.
-This is the same as turning off the power, which means that data loss may occur.
+ -- TurnOff: Turns off the virtual machine without shutting down the operating system first, then
+ migrates the virtual machine. This is the same as turning off the power, which means that data loss
+ may occur.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md:177]:
```suggestion
- `TurnOff`: Turns off the virtual machine without shutting down the operating system first, then
  migrates the virtual machine. This is the same as turning off the power, which means that data
  loss may occur.
```

File: docset/winserver2022-ps/failoverclusters/New-Cluster.md [hunk 6]
```diff
@@ -104,25 +112,26 @@ Name
...
-The cmdlet creates an administrative access point for the cluster.
-The administrative access point is registered in DNS but is not enabled in Active Directory Domain Services.
+
+Specifies the type of administrative access point that the cmdlet creates for the cluster. The
+acceptable values for this parameter are:
+
+- ActiveDirectoryAndDns. The cmdlet creates an administrative access point for the cluster. The
+  administrative access point is registered in DNS and enabled in Active Directory Domain Services.
+- Dns. The cmdlet creates an administrative access point for the cluster. The administrative access
+  point is registered in DNS but is not enabled in Active Directory Domain Services.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/New-Cluster.md:128]: To keep the in-console help from breaking, we need blank lines between list items in parameters.

```suggestion
- ActiveDirectoryAndDns. The cmdlet creates an administrative access point for the cluster. The
  administrative access point is registered in DNS and enabled in Active Directory Domain Services.

- Dns. The cmdlet creates an administrative access point for the cluster. The administrative access
  point is registered in DNS but is not enabled in Active Directory Domain Services.

```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Minor notes for syntax that breaks in-console help.

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**